package.json

```diff
@@ -41,7 +41,6 @@
     "applicationinsights": "^0.19.0",
     "chalk": "^1.1.3",
     "commander": "^2.9.0",
-    "node-status": "^1.0.0",
```

Comment:
I think this may have already been documented as a known issue in this package. @nokafor , are you OK with this approach?